Assets/Stack mini game/Scripts/ChangeMovingCubeDirectionCollider.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ChangeMovingCubeDirectionCollider : MonoBehaviour
+{
+    public Action collisionWithMovingCubeEvent;

```

Comment:
Naming violation here. Should be PascalCase as it's a public field.

Reply:
```CollisionWithMovingCubeEvent```

Reply:
Done